compiler/lib/js_output.ml

```diff
@@ -1996,25 +2149,52 @@ let program ?(accept_unnamed_var = false) f ?source_map p =
               | None -> filename
               | Some _ -> Filename.concat "/builtin" filename)
         in
+        let ignore_list =
+          List.filter sources ~f:(fun filename ->
+              String.length filename >= 9
+              && String.equal (String.sub filename ~pos:0 ~len:9) "/builtin/")
```

Comment:
String.starts_with

Reply:
This is only available since OCaml 3.13. But maybe we should add it to `stdlib.ml`?

Reply:
We have String.is_prefix in our stdlib